product_docs/docs/epas/17/epas_rel_notes/epas17_2_rel_notes.mdx

```diff
@@ -0,0 +1,30 @@
+
+!!! Note Deprecation
+With the release of EPAS 17, the DRITA is being deprecated and will not be included in EPAS 18. This decision follows the introduction of [EDB’s PWR](/pwr/latest/) and [`edb_wait_states`](/pg_extensions/wait_states/) extensions, which offer improved diagnostic capabilities compared to DRITA. These new extensions provide a diagnostic experience more closely aligned with Oracle’s AWR reports, making DRITA obsolete.
+!!!
+
+| Type           | Description                                                                                                                                                                                                                                                                              | Category |
+|----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|
+| Upstream merge | Merged with community PostgreSQL 17.2. See the [PostgreSQL 17 Release Notes](https://www.postgresql.org/docs/17/release-17-2.html) for more information.                                                                                                                                 |          |
+| Feature        | Added support for the Oracle-compatible `BFILE` native datatype and the `DBMS_LOB` package APIs. See the [DBMS_LOB](../reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/06_dbms_lob/) for more information.                                           |          |
+| Feature        | Added support for the Oracle-compatible `DBMS_XMLDOM` package to provide interface for HTML and XML documents. See the [DBMS_XMLDOM](../reference/oracle_compatibility_reference/epas_compat_bip_guide/dbms_xmldom) for more information.                                                |          |
+| Feature        | Added support for the Oracle-compatible `DBMS_ASSERT` package validate input properties and sanitize user input, thereby reducing the risk of SQL injections. See the [DBMS_ASSERT](../reference/oracle_compatibility_reference/epas_compat_bip_guide/dbms_assert) for more information. |          |
+| Feature        | Added support for the Oracle-equivalent `NLS_UPPER`,`NLS_LOWER`, and `NLS_INITCAP` functions. See the [NLS functions](../reference/sql_reference/03_functions_and_operators/nls_functions) for more information.                                                                         |          |
+| Feature        | Implemented `alteruser` utility to modify roles in the clusters. See the [alteruser utility](/docs/tools/alteruser_utility/) for more information.                                                                                                                                       |          |
```

Comment:
Not sure if this will fix the broken link. Let's check after committing.
```suggestion
| Feature        | Implemented `alteruser` utility to modify roles in the clusters. See the [alteruser utility](/tools/alteruser_utility/) for more information.                                                                                                                                       |          |
```